apis/kueue/v1beta1/localqueue_types.go

```diff
@@ -73,6 +73,32 @@ type LocalQueueFlavorStatus struct {
 	// +kubebuilder:validation:MaxItems=8
 	// +optional
 	NodeTaints []corev1.Taint `json:"nodeTaints,omitempty"`
+
+	// topology is the topology that associated with this ResourceFlavor.
+	//
+	// This is an alpha field and requires enabling the TopologyAwareScheduling
+	// feature gate.
+	//
+	// +featureGate=TopologyAwareScheduling
+	// +optional
+	Topology *Topology `json:"topology,omitempty"`
```

Comment:
Why is this needed? The info is available in ResourceFlavor which is referenced couple fields above.

Reply:
ResourceFlavors and Topologies are cluster-scoped resources, and are not visible to end-users. 

This is basically the same motivation as for the KEP which introduced LocalQueueFlavorStatus: https://github.com/kubernetes-sigs/kueue/issues/3122

Reply:
I understand that part. While I understand why ClusterQueues may not be readable for users, i don't see any point in hiding ResourceFlavors. That builds more features in this weird design where we dump spec of one object into the status of the other one. 

While, for the sake of completeness and consistency with the previous decisions, we may accept the change,  I would propose to take a really close look at this part of the api when working on v1beta2 or GA.

Reply:
I agree it is worth re-visiting the topic and alternative approaches might be possible, like a dedicated LQ subresource. However, I'm leaning to accept the change as is for the sake of completeness and providing the information to the end-users in 0.11.